src/lib.rs

```diff
@@ -734,29 +768,29 @@ impl<V: Clone> Clone for VecMap<V> {
     }
 }
 
-impl<V: PartialEq> PartialEq for VecMap<V> {
```

Comment:
What's the story for comparison with regard to allocators?

Reply:
After looking at the `Vec` in the standard library, it seems as that `PartialEq` and `PartialOrd` can be implemented for different allocators. I tried and it works so I will make a new pr with these changes.